### ISSUE TITLE
bad Galaxy score because of whitespace warnings

### ISSUE THREAD

**Issue by pcahyna** [linux-system-roles/timesync#56]
Ansible Galaxy gives timesync a quality score of 2.5/5 solely because of whitespace warnings in tests. The import page says:
```
===== LINTING ROLE =====
 yamllint Warnings:
 ./tests/tests_default.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp_provider1.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp_provider2.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp_provider3.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp_provider4.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp_provider5.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp_provider6.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ntp_ptp.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ptp_multi.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
 ./tests/tests_ptp_single.yml:1:1: [error] too many blank lines (1 > 0) (empty-lines)
```
Tests are not checked by Molecule which otherwise does a yamllint pass over the role, so this did not get caught.

**Comment by pcahyna** [comment]
> Tests are not checked by Molecule which otherwise does a yamllint pass over the role, so this did not get caught.

Maybe this would be solved by https://github.com/linux-system-roles/network/pull/170 ?

**Comment by pcahyna** [comment]
it got merged into the template as https://github.com/linux-system-roles/template/pull/15

**Comment by pcahyna** [comment]
it works, as shown by https://travis-ci.org/github/linux-system-roles/timesync/jobs/667204606

**Comment by richm** [comment]
I think this is fixed by https://github.com/linux-system-roles/timesync/pull/67 - please confirm - if so, let's close this PR in favor of https://github.com/linux-system-roles/timesync/pull/67

**Comment by pcahyna** [comment]
I believe so, if you are calling yamllint the new way from the template, it catches the same "problem" as Galaxy import, I tested that in #66

**Comment by richm** [comment]
closed in favor of #67

**Comment by pcahyna** [comment]
Ah, this is an issue - I think it is better to close issues only when the PRs fixing them are merged - GitHub can do this automatically for you.
